test/Microsoft.Extensions.FileSystemGlobbing.Tests/FunctionalTests.cs

```diff
@@ -382,7 +383,8 @@ public void StemCorrectWithDifferentWildCards_WithInMemory()
             var results = matcher.Execute(new InMemoryDirectoryInfo(directoryPath, files));
```

Comment:
Do you want to use the new extension methods here instead? Just so that the new code path is also exercised?